### ISSUE TITLE
Build error currently showing

### ISSUE THREAD

**Issue by mariobehling** [loklak/loklak_server#1226]
Right now there is a build error of the app. Please help to fix that.

@SarishiNoHara @djmgit @kavithaenair @vibhcool @Achint08 @hemantjadon @singhpratyush @SKrPl

**Comment by vibhcool** [comment]
there is some permission error for deployment of the project.
```
https://travis-ci.org/loklak/loklak_server/builds/239267625#L1342
```

**Comment by vibhcool** [comment]
project is building successfully

**Comment by singhpratyush** [comment]
The error is in deployment part of travis. Here is the error message - 
```
'raw/server' does not exist; use 'git subtree add'
git fetch ../loklak_server documentation
fatal: '../loklak_server' does not appear to be a git repository
fatal: Could not read from remote repository.
```

It is related to issues #1215 #1204 and PRs #1207 #1195.

@imujjwal96: Could you please help us with this?

**Comment by imujjwal96** [comment]
@pri22296  @sch00lb0y  Can you please have a look why the build is still failing? I am not able to find a solution for this.

**Comment by pri22296** [comment]
@imujjwal96 I tested this locally. subtree logic seems to be fine. I think the issue is with the permission. do we have the keys setup to push to dev.loklak.org repository? Also it would be helpful to debug if the push doesn't happen quietly.

**Comment by singhpratyush** [comment]
@mariobehling: This can be closed as it was fixed in cd04d1940e7c70f67a021ea794e9332cef136b8d.
